tests/core/atoms/test_skzcam.py

```diff
@@ -30,11 +36,984 @@ def embedded_cluster():
+            '"aug-cc-pVDZ/C"',
+            "end",
+            "HFTyp",
+            "Guess",
+            "MOInp",
+            "SCFMode",
+            "sthresh",
+            "AutoTRAHIter",
+            "MaxIter",
+            "end",
+            "CTyp",
+            "Mult",
+            "Units",
+            "Charge",
+            "coords",
+            "end",
+        ]
+    )
+
+    # Check that the strings and floats in slab_coords matches reference
+    assert_allclose(
+        [
+            float(x)
+            for x in slab_block.split()[::10]
+            if x.replace(".", "", 1).replace("-", "", 1).isdigit()
+        ],
+        [
+            3.128,
+            0.0,
+            0.00567209089,
+            2.0,
+            1.203,
+            5.1757,
+            1.0,
+            2.0,
+            1.203,
+            5.1757,
+            1.0,
+            2.0,
+            1.203,
+            5.1757,
+            1.0,
+            2.0,
+            1.203,
+            5.1757,
+            1.0,
+            2.0,
+            1.203,
+            5.1757,
+            1.0,
+            2.0,
+            1.203,
+            5.1757,
+            1.0,
+            2.0,
+            1.203,
+        ],
+        rtol=1e-05,
+        atol=1e-07,
+    )
+
+    assert np.all(
+        [
+            x
+            for x in slab_block.split()[::5]
+            if not x.replace(".", "", 1).replace("-", "", 1).isdigit()
+        ]
+        == [
+            "%pal",
+            "Method",
+            "Energy",
+            "NewNCore",
+            "O",
+            "NewGTO",
+            "Mg",
+            '"aug-cc-pVDZ"',
+            "end",
+            "NewAuxJGTO",
+            "C",
+            '"cc-pVDZ/C"',
+            "end",
+            "Guess",
+            "Direct",
+            "MaxIter",
+            "xyz",
+            "Charge",
+            "O",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "Mg>",
+            "NewECP",
+            "s",
+            "end",
+        ]
+    )
+
+
+def test_create_atom_coord_string(embedded_adsorbed_cluster):
+    atom = embedded_adsorbed_cluster[0]
+
+    # First let's try the case where it's a normal atom.
+    atom_coord_string = create_atom_coord_string(atom)
+
+    with pytest.raises(ValueError):
```

Comment:
Here and below, you should specify a `match` pattern in the command, e.g. `pytest.raises(ValueError, match="the string I want to capture")`.